Lib/copy.py

```diff
@@ -238,6 +238,11 @@ def _deepcopy_method(x, memo): # Copy instance methods
     return type(x)(x.__func__, deepcopy(x.__self__, memo))
 d[types.MethodType] = _deepcopy_method
 
+def _deepcopy_memoryview(x, memo, deepcopy=deepcopy):
+    y = memoryview(deepcopy(x.obj, memo))
```

Comment:
This can fail in several ways:

- Doesn't preserver readonly state. For example, copying `memoryview(bytearray(b"hi")).toreadonly()` will create an MV that's not readonly. This is fixable though.
- Memoryviews created through `PyMemoryView_FromBuffer` or  `PyMemoryView_FromMemory` may not have an underlying `obj`, and cannot be constructed directly from Python, so this will not copy them faithfully. I don't see a way to fix this.